util/env_posix.cc

```diff
@@ -675,6 +675,10 @@ class PosixEnv : public Env {
 
   virtual void SleepForMicroseconds(int micros) override { usleep(micros); }
 
+  virtual void SleepForNanoseconds(int nanos) override {
```

Comment:
We can do that. The only concern is that reading the code will be confusing, just like how @yuslepukhin was confused.